### DISCUSSION TITLE
Statistic 模版代码拷贝到本地 报错

### DISCUSSION THREAD

**Discussion by xiaoyuanxun** [ant-design/ant-design#44973] [Q&A] [answered]
[Statistic统计数值](https://ant.design/components/statistic-cn#api)的这个[组件](https://ant.design/~demos/statistic-demo-animated)，我将模版代码拷贝到本地的一个react项目的compoents文件夹中，但是编译报错。

![截屏2023-09-20 下午4 25 29](https://github.com/ant-design/ant-design/assets/53613219/b1b08bd7-d37f-4887-b39b-381f29fb87fe)

![截屏2023-09-20 下午4 28 07](https://github.com/ant-design/ant-design/assets/53613219/089c062b-4fa2-4a5c-8600-e511d9923fb1)

请问如何解决？

**Comment by MadCcc** [comment, marked as answer]
typescript 开了 strict 吧

**Reply by xiaoyuanxun** [reply]
谢谢。
我将tsconfig.json中的    "strict": true 取消后， 报错没有了。
